sdk/src/client/secret/mnemonic.rs

```diff
@@ -121,17 +122,17 @@ impl SecretManage for MnemonicSecretManager {
         Ok((public_key, signature))
     }
 
-    async fn sign_transaction_essence(
+    async fn transaction_unlocks(
```

Comment:
Should this be part of the Trait? Seems kind of weird. Maybe we should just remove both.

Reply:
Yeah maybe, they still have different behaviour depending on the impl but I'll create an issue to reconsider what should be part of the trait or not